metadata.yaml

```
@@ -0,0 +1,8 @@
+name: sssd-operator
+display-name: sssd-operator
```

Should the display name be "SSSD Client"?

```suggestion
display-name: SSSD Client
```

---

Sounds good, updated!